Organization/index.ts

```diff
@@ -4,11 +4,12 @@ import { Email } from "../Email"
 import { Changeable as OrganizationChangeable } from "./Changeable"
 import { Creatable as OrganizationCreatable } from "./Creatable"
 import { Identifier as OrganizationIdentifier } from "./Identifier"
-export interface Organization extends Omit<Organization.Creatable, "users"> {
+export interface Organization extends Omit<Organization.Creatable, "users" | "permissions"> {
```

Comment:
for my own education, what happens here?

Reply:
Organization is of a type equal to Creatable, but it removes the properties stated "users" | " permissions"

Reply:
Then we add them again, but with different types compared to in Creatable. 

You can read more here: https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys